<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriorityLayerDialog</class>
 <widget class="QDialog" name="PriorityLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Priority Layer Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Priority layer name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="layer_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Priority layer description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="layer_description"/>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="group_box">
     <property name="title">
      <string>Priority groups</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Carbon importance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSpinBox" name="carbon_importance_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Biodiversity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsSpinBox" name="biodiversity_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Livelihood</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsSpinBox" name="livelihood_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Climate Resilience</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsSpinBox" name="climate_resilience_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ecological infrastructure</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QgsSpinBox" name="ecological_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Policy</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QgsSpinBox" name="policy_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Finance - Years Experience</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QgsSpinBox" name="finance_experience_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Finance - Market Trends</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QgsSpinBox" name="finance_market_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Finance - Net Present value</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QgsSpinBox" name="finance_present_value_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Finance - Carbon</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QgsSpinBox" name="finance_carbon_box">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>PriorityLayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>PriorityLayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
